liblsd/hostlist.c

```diff
@@ -381,7 +381,7 @@ static char * _next_tok(char *sep, char **str)
     char *tok;
 
     /* push str past any leading separators */
-    while (**str != '\0' && strchr(sep, **str) != '\0')
+    while (**str != '\0' && strchr(sep, **str) != NULL)
```

Comment:
Do you want only hostlist.c, or the whole first commit?

Reply:
All the changes in this commit are good - just wanted to split it into two commits, one that applies the changes to hostlist.c.  (We use that module in other projects so it would be nice to be able to refer to a standalone commit that improves it).

Thanks!